packages/cli/src/metadataGeneration/initializer-value.ts

```diff
@@ -39,6 +39,8 @@ export const getInitializerValue = (initializer?: ts.Expression, typeChecker?: t
         return dateString;
       }
       return;
+    case ts.SyntaxKind.NullKeyword:
+      return null;
```

Comment:
Without this case, `null` would fall to the `default` case, being returned as `undefined`.